src/UI/templates/default/Input/tpl.context-form.html

```diff
@@ -1,5 +1,5 @@
 <div class="form-group row">
-	<label for="{NAME}" class="control-label col-sm-3">{LABEL}</label>
+	<label for="{NAME}" class="control-label col-sm-3">{LABEL}<!-- BEGIN required --><span class="asterisk">*</span><!-- END required --></label>
```

Comment:
This seems to be wrong (although it currently is done in that exact way). Shouldn't some upper element like the div get a class="required" or something similar and this span be included by CSS?